tests/index.test.ts

```diff
@@ -349,3 +349,34 @@ test(`create`, async t => {
     });
   });
 });
+
+test(`body not parsed with flush`, async t => {
+  const server = createServer((req, res) => {
+    const body = { message: "foo" };
+    res.writeHead(200, {
+      "Content-Type": "application/json",
+      "Content-Length": JSON.stringify(body).length
+    });
+    res.end(JSON.stringify(body));
+  });
+
+  await new Promise(r => {
+    server.listen(async () => {
+      // @ts-ignore
+      const { port } = server.address();
+
+      const res = await gretch(`http://127.0.0.1:${port}`).flush();
```

Comment:
I would add a test for the `url/response/status` here?

Reply:
For sure, will add that rn.